frontend/src/components/common/form/StepForm/SteppedForm.tsx

```diff
@@ -92,11 +99,18 @@ export const SteppedForm = function<T extends object = {}>({
   if (!getTabs) {
     initialValues.tabs = [{ activeStep: initialValues.activeStep }];
   }
+  if (!initialValues.tabs && !!getTabs) {
```

Comment:
set initial tabs in initial values to prevent false positive checks on touched and dirty formik props.